arviz/plots/backends/matplotlib/traceplot.py

```diff
@@ -193,9 +203,9 @@ def plot_trace(
     plot_kwargs.setdefault("linewidth", linewidth)
 
     if axes is None:
-        _, axes = plt.subplots(len(plotters), 2, squeeze=False, figsize=figsize, **backend_kwargs)
+        fig = plt.figure(tight_layout=True, figsize=figsize)
```

Comment:
We should not use `tight_layout` as is not compatible with `constrained_layout=True` that is the arviz's default.